src/components/Link/Contact.tsx

```diff
@@ -0,0 +1,125 @@
+import {
+  useDeskproAppEvents,
+  useInitialisedDeskproAppClient,
+  useQueryWithClient,
+} from "@deskpro/app-sdk";
+import { AnyIcon, Button, Checkbox, Input, Stack } from "@deskpro/deskpro-ui";
+import { faMagnifyingGlass } from "@fortawesome/free-solid-svg-icons";
+import { useState } from "react";
+import { useNavigate } from "react-router-dom";
+import useDebounce from "../../hooks/debounce";
+import { useLinkContact } from "../../hooks/hooks";
+import ContactJson from "../../mapping/contact.json";
+import { Title } from "../../styles";
+import { FieldMapping } from "../FieldMapping/FieldMapping";
+import { HorizontalDivider } from "../HorizontalDivider/HorizontalDivider";
+import { LoadingSpinnerCenter } from "../LoadingSpinnerCenter/LoadingSpinnerCenter";
+import { getContactsByEmail } from "../../api/api";
+
+export const LinkContact = () => {
+  const [selectedContact, setSelectedContact] = useState<string | null>(null);
+  const [prompt, setPrompt] = useState<string>("");
+  const { linkContact } = useLinkContact();
+  const navigate = useNavigate();
+
+  const { debouncedValue: debouncedText } = useDebounce(prompt, 300);
+
+  useInitialisedDeskproAppClient((client) => {
+    client.setTitle("Link Contact");
+
+    client.registerElement("homeButton", {
+      type: "home_button",
+    });
+
+    client.deregisterElement("plusButton");
+
+    client.deregisterElement("menuButton");
+  }, []);
+
+  useDeskproAppEvents({
+    async onElementEvent(id) {
+      switch (id) {
+        case "homeButton":
+          navigate("/redirect");
+      }
+    },
+  });
+
+  const contactsQuery = useQueryWithClient(
+    ["getContactsByEmail", debouncedText],
+    (client) => getContactsByEmail(client, debouncedText),
+    {
+      enabled: debouncedText.length > 2,
+    }
+  );
+
+  const contacts = contactsQuery.data?.value;
+
+  return (
+    <Stack gap={10} style={{ width: "100%" }} vertical>
+      <Stack vertical gap={6} style={{ width: "100%" }}>
+        <Input
```

Comment:
use `Search` component from `app-sdk`

Reply:
The input component looks alot better 
![image](https://github.com/DeskproApps/dynamics365/assets/54243490/e073df3e-c319-4fd7-9e4a-18f319a1ef51)
![image](https://github.com/DeskproApps/dynamics365/assets/54243490/fa2a2607-8839-4cbc-a900-fb47971fa331)

Reply:
ok